### PR TITLE
1️⃣Update status demo view to match the current webhook posted data

### PR DESCRIPTION
**Pull request by norkans7**
Fix the sample flow broken since the webhook always return invalid order ID

### CODE REVIEW

File: temba/public/tests.py [hunk 1]
```diff
@@ -97,23 +99,31 @@ def test_demo_status(self):
         self.assertEqual(response.request["PATH_INFO"], status_url)
         self.assertContains(response, "Cancelled")
 
-        response = self.client.post(status_url, {}, follow=True)
+        response = self.client.post(status_url, json.dumps({}), content_type="application/json", follow=True)
```
Comment (rowanseymour) [inline temba/public/tests.py]: pretty sure the Django client takes care of encoding/dumping when `content_type=="application/json"` so you just have to write...

```
self.client.post(status_url, {}, content_type="application/json", follow=True)
```

**Comment by nicpottier** [comment]
What has changed here? Not sure I understand what is different. Shouldn't the new engine be posting the same URL as before?

**Comment by nicpottier** [comment]
Oh, this has been broken forever, since we changed our webhook payload format a year or more ago. Well shoot.

**Comment by norkans7** [comment]
Yep

**Pull request merged**